docs/dev/annual_updates.rst

```diff
@@ -80,31 +89,41 @@ the years (e.g. ``boiler_fuel``). However ``page_name`` does not necessarily cor
 directly to PUDL database table names because we don't load the data from all pages, and
 some pages result in more than one database table after normalization.
 
+**2.1)** Add a column for the new year of data to each of the aforementioned files. If
+there are any changes too prior years, make sure to address those too. (See note above).
+If you are updating early release data with final release data, replace the values in
+the appropriate year column.
+
 .. note::
 
-    As mentioned above, sometimes EIA will change files published several years ago
-    without providing any explanation. When creating new raw input archives for Zenodo,
-    note which years of data have been altered so you can be particularly alert to
-    changes in those files.
+   If you are adding EIA's early release data, make sure the raw files have
+   ``Early_Release`` at the end of the file name. This is how the excel extractor knows
+   to label the data as provisional vs. final.
+
+   Early release files also tend to have one extra row at the top and one extra column
+   on the right of each file indicating that it is early release. This means that the
+   skiprows and column map values will probably be off by 1 when you update from early
+   release to final release.
+
+**2.2)** If there are files, spreadsheet pages, or individual columns with new semantic
+meaning (i.e. they don't correspond to any of the previously mapped files,
+pages, or columns) then create new mappings to track that information over time.
 
-If files, spreadsheet pages, or individual columns with new semantic meanings have
-appeared -- meaning they don’t correspond to any of the previously mapped files, pages,
-or columns, then new mappings analogous to the above need to be created to track that
-information over time.
+.. note::
 
-In all of the the above CSV files we use a value of ``-1`` to indicate that the data
-does not exist in a given year.
+    In all of the the above CSV files we use a value of ``-1`` to indicate that the data
+    does not exist in a given year.
 
 FERC Form 1
 ^^^^^^^^^^^
-The path to the directory containing the database files stored within the annual FERC 1
-zipfiles changes from year to year, and will need to be updated for the new year of
-data. We store this information in ``src/pudl/package_data/ferc1/file_map.csv``
+**2.1)** Update the path to the directory containing the database files stored within
```

Comment:
i'm not sure if this really deeply matters but it feels weird to have many instances of `2.1` in here for every data source. i think if we are going to stick to the numbered bit (which i like!) we should also number the data sources.

Reply:
Ok! I was also uncertain about this b/c I didn't want to implicate that one had to go through all the steps if you were only updating one of the data sources.